### ISSUE TITLE
[Feature] support for general logit processors

### ISSUE THREAD

**Issue by lbeurerkellner** [huggingface/text-generation-inference#240]
Are there any plans to support logit processors via an additional API parameters? For instance, the OpenAI API provides a `logit_bias` parameter, that is applied to the token distribution during generation.

This would be great for http://github.com/eth-sri/lmql, where we heavily rely on token masking to enforce constraints during decoding.

Thank you for your hard work on this project.

**Comment by Vinno97** [comment]
Also very interested in this. For tasks like classification, it can be really useful to constrain the output to just some select logits. Especially when you want to have a "confidence" calculation

**Comment by Vinno97** [comment]
🤗 transformers just landed the [SequenceBiasLogitsProcessor](https://huggingface.co/docs/transformers/main/en/internal/generation_utils#transformers.SequenceBiasLogitsProcessor) in their main branch, meaning this should be fairly straightforward to also implement here now as well!

**Comment by AlvinNg89** [comment]
Hi, am interested in this as well. Any chance of this being implemented?
